tembo-cli/src/cli/context.rs

```diff
@@ -88,6 +88,14 @@ pub fn tembo_credentials_file_path() -> String {
     tembo_home_dir() + "/credentials"
 }
 
+pub fn dot_tembo_folder() -> String {
+    ".tembo".to_string()
```

Comment:
Cant these just be a `const DOT_TEMBO_FOLDER: String = String::from(".tembo.io")`?

Reply:
Yup, good catch. Changed it.